### PR TITLE
update and rename libstatic playbook

### PR DESCRIPTION
**Pull request by VickieKarasic**
Partial fix for #2852

### CODE REVIEW

File: playbooks/libstatic.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+# by default this playbook runs in the staging environment
+# to run in qa, pass '-e runtime_env=qa'
```
Comment (sandbergja) [inline playbooks/libstatic.yml:3]: There is no qa environment for libstatic.

**Review by sandbergja — CHANGES_REQUESTED** [review]
Thanks, @VickieKarasic !  It will be really nice to have the unified playbook and the improved slack message!

I'm not sure this will work with the production `runtime_env`.  Currently, the production playbook specifies a separate `vars_file` and a dependency on the datadog role.  Would you be able to bring those into this playbook as well, and delete the old `libstatic_prod.yml` playbook as part of this PR?

File: playbooks/libstatic.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# to run in production, pass '-e runtime_env=production'
+- name: Build libstatic infrastructure
+  hosts: libstatic_{{ runtime_env | default('staging') }}
+  remote_user: pulsys
+  become: true
+  roles:
+    - role: libstatic
+      when: runtime_env | default('staging') == "production"
```
Comment (sandbergja) [inline playbooks/libstatic.yml:11]: Do we need this `when`?  I think this playbook should always run the `libstatic` role.

File: playbooks/libstatic.yml (hunk 1 above)
Comment (VickieKarasic) [reply playbooks/libstatic.yml]: Thanks, @sandbergja! I removed that comment from the playbook.

File: playbooks/libstatic.yml (hunk 2 above)
Comment (VickieKarasic) [reply playbooks/libstatic.yml:13]: You're right! @acozine and I added the datadog role and moved the `when` to that.

**Comment by VickieKarasic** [comment]
Thanks for all these helpful changes, @sandbergja! I think I caught all of them, if you'd like to take another look when you get a chance.

**Review by sandbergja — APPROVED** [review]
This looks awesome!  Thanks, @VickieKarasic ! 🎉

**Pull request merged**